### DISCUSSION TITLE
css doesn't work

### DISCUSSION THREAD

**Discussion by AmandaSpinella** [community/community#108440] [Programming Help]
### Body

My css isn't working on my page, but I don't know what I did wrong.
https://github.com/AmandaSpinella/AmandaSpinella.github.io

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ghost** [comment]
Sorry but the link you have mentioned is not found 404. Try to check on that....



Also if the CSS is not working...try these

- Check whether you are linking the correct css file.
- Clear your browser cache.
- Remember files are case sensitive.
- Also check whether your css doesn't have any syntactic issues or error.

**Reply by ghost** [reply]
Hey hi @AmandaSpinella your project works really awesome both locally and in deployed stage. Guess you had fixed it. Anyways all the best in your web dev journey....

Thank you.

**Comment by ottogugel** [comment]
I can't access your project page, but are you doing a project with HTML and CSS?

If so, how did you structure it? Did you create an example tag ```<h1>Test </h1> ``` and assign the class/id/tag to the tag? ``` <h1 class=""> Test </h1> ```

Remember that you have to see what the selector is as id you put #id if it is class just put class="id"
and if it's the tag itself, just put its name in the .css.

**Comment by albedim** [comment]
Hi, i tried to clone your repo and it works perfectly for me

**Comment by AmandaSpinella** [comment]
I had found the source of the problem. Thanks for the feedback.


Em dom., 18 de fev. de 2024 às 10:05, Yuhesh Pandian P J <
***@***.***> escreveu:

**Reply by ghost** [reply]
Your welcome 😎

**Comment by AmandaSpinella** [comment]
Now there's just one problem. I couldn't make the "Formation" part well
structured for smartphones.


Em dom., 18 de fev. de 2024 às 10:09, Yuhesh Pandian P J <
***@***.***> escreveu:

**Reply by ghost** [reply]
That is because there is some contents that overflow to cause in that scroll behaviour on x axis

**Reply by ghost** [reply]
Try reducing the space between the contents that cause the overflow

**Reply by ghost** [reply]
Use dev tools to find really causing the issue

**Comment by Bl4ck-H4ck3r** [comment]
This project works properly

**Comment by niladri-1** [comment]
Hi, @AmandaSpinella 

Web site working fine in my computer,
I found a syntax mistack in `formacao.css` file , at the last add a `"}"`


```formacao.css
@media (max-width: 800px) {
  .formacao-container {
    grid-template-columns: 1fr;
    padding: 60px 20px;
    gap: 40px;
  }
  .formacao .subtitulo {
    color: #ffffff;
  }
  .formacao-texto {
    font-size: 1.25rem;
  }
  .formacao-extra h3::before {
    position: initial;
    margin-bottom: 10px;
  }
}
```
